src/Pages/HomePage/HomePage.css

```diff
@@ -0,0 +1,56 @@
+:root{
+    --bg-color: #fcfcfc;
+    --category-color:#e7e2e2;
+    --border-color : #a3a2a2;
+}
+
+.page-container{
+    display: flex;
+    text-align: initial;
+    margin-top: 4rem;
+    width: 100vw;
+}
+
+.home-section{
+    width: 100vw;
+}
+
+.videos-container{
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+    gap: 1rem 1rem;
+    background-color: var(--bg-color);
+    padding: 1rem;
+}
+
+.video{
+   background-color: var(--white-color);
+   max-width: 300px;
+   max-height: auto;
+   gap: 1rem;
+   cursor: pointer;
+   height: 320px;
```

Comment:
try to use rems instead of px

Reply:
thanks for the suggestion, changing px to rem.